types/database/user.d.ts

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import Model, {Associations} from '@nozbe/watermelondb/Model';
+
+import Channel from '@typings/database/channel';
+import ChannelMembership from '@typings/database/channel_membership';
+import GroupMembership from '@typings/database/group_membership';
+import Post from '@typings/database/post';
+import Preference from '@typings/database/preference';
+import Reaction from '@typings/database/reaction';
+import TeamMembership from '@typings/database/team_membership';
+
+/**
+ * The User model represents the 'USER' entity and its relationship to other
+ * shareholders in the app.
+ */
+export default class User extends Model {
+    /** table (entity name) : User */
+    static table: string;
+
+    /** associations : Describes every relationship to this entity. */
+    static associations: Associations;
+
+    /** auth_service : The type of authentication service registered to that user */
+    authService: string;
+
+    /** delete_at : The timestamp at which this user account has been archived/deleted */
+    deleteAt: number;
+
+    /** email : The email address for that user  */
+    email: string;
+
+    /** first_name : The user's first name */
+    firstName: string;
+
+    /** is_bot : Boolean flag indicating if the user is a bot */
+    isBot: boolean;
+
+    /** is_guest : Boolean flag indicating if the user is a guest */
+    isGuest: boolean;
+
+    /** last_name : The user's last name */
+    lastName: string;
+
+    /** last_picture_update : The timestamp of the last time the profile picture has been updated */
+    lastPictureUpdate: number;
+
+    /** locale : The user's locale */
+    locale: string;
+
+    /** nick_name : The user's nick name */
```

Comment:
```suggestion
    /** nickname : The user's nickname */
```

nickname is just one word, right ? 👈 @migbot  ?

Reply:
It is `nickname` under the mm-redux entities

Reply:
yeap, and is `nickname` in the server database & in the diagram we created